ui/src/css/components/_RepoHeader.scss

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#repoheader {
+  border-bottom: 1px solid #dadce0;
+  overflow: auto;
+}
+
+#repoheader-backbutton {
+  float: left;
```

Comment:
Optional: Flexbox may be a better alternative (i.e., with less tricky behavior) to float for this kind of thing.

Reply:
Thanks. I'll consider it when we get the details of the spacing and stuff here nailed down.